introduction.md

```diff
@@ -1,3 +1,7 @@
 Hi, my name is what? My name is, who? My name is jikka jikka jikka Slim Shady.
 
 My favorite Code School path is JavaScript
+
+My deadly skills are 
+* C# 
+* Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?